src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,134 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.ExitCommand;
+import duke.command.AddCommand;
+import duke.command.DoneCommand;
+import duke.command.DeleteCommand;
+import duke.command.InputCommand;
+import duke.command.GetCommand;
+import duke.command.ListCommand;
+import duke.command.FindCommand;
```

Comment:
It is good that you group the import statements consistently. However (this is just nitpicking, not compulsory), it looks neater if you write them in lexicographical order.